apiserver/facades/client/client/package_test.go

```diff
@@ -9,8 +9,8 @@ import (
 	coretesting "github.com/juju/juju/internal/testing"
 )
 
-//go:generate go run go.uber.org/mock/mockgen -typed -package client_test -destination package_mock_test.go github.com/juju/juju/apiserver/facades/client/client Backend,Model
-//go:generate go run go.uber.org/mock/mockgen -package client_test -destination service_mock_test.go github.com/juju/juju/apiserver/facades/client/client BlockDeviceService,NetworkService
+//go:generate go run go.uber.org/mock/mockgen -typed -package client_test -destination package_mock_test.go github.com/juju/juju/apiserver/facades/client/client Backend
+//go:generate go run go.uber.org/mock/mockgen -package client -destination service_mock_test.go github.com/juju/juju/apiserver/facades/client/client BlockDeviceService,NetworkService,ModelInfoService
```

Comment:
The mocks can stay in the client_test package.
You can change the package of this file to client and add a helper to do 
`client := &Client{modelInfoService: s.modelInfoService}`

Reply:
I thought we now prefer not to use the test package for test and mocks anymore.